src/Farmer/Arm/Storage.fs

```diff
@@ -24,7 +25,24 @@ type StorageAccount =
                apiVersion = "2018-07-01"
                location = this.Location.ArmValue
             |} :> _
-
+    interface IPostDeploy with
+        member this.Run resourceGroupName =
+            match this with
+            | { StaticWebsite = Some (indexDoc, errorDoc, folder); Name = name } ->
+                printfn "Enabling static web site for storage account %s with Index document as %s, Error document as %s" name.Value indexDoc errorDoc 
+                Deploy.Az.enableStaticWebsite name.Value indexDoc errorDoc
```

Comment:
I wonder if this could be enabled via the ARM template itself?

Reply:
I tried to google + export template from existing static website, but no luck. Also found few mentions that it's not supported for ARM out of the box.

Reply:
Got "inspired" 😉 by this post: http://www.frankysnotes.com/2019/03/deploy-automatically-static-website.html

Reply:
Yeah just saw - a dead end.